frontend/src/libs/components/header/styles.module.scss

```
@@ -1,8 +1,17 @@
+.container {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  z-index: var(--z-index-level-one);
+}
+
 .header {
   position: relative;
   display: flex;
   justify-content: space-between;
   align-items: center;
+  height: var(--header-height);
   padding: 30px 60px;
   background: var(--light-grey);
   border-bottom: 2px solid var(--light-grey-15);
```

get rid of `div` tag and container class.
and it will be enough to add styles to header

```suggestion
.header {
  position: fixed;
  z-index: var(--z-index-level-one);
  display: flex;
  justify-content: space-between;
  align-items: center;
  width: 100%;
  height: var(--header-height);
  padding: 30px 60px;
  background: var(--light-grey);
  border-bottom: 2px solid var(--light-grey-15);
}
```